src/lib/viewers/__tests__/BaseViewer-test.js

```diff
@@ -1254,6 +1254,27 @@ describe('lib/viewers/BaseViewer', () => {
         });
     });
 
+    describe('areNewAnnotationsEnabled()', () => {
+        it('should return false if the file is excel or iWork formats', () => {
```

Comment:
Should we have a parameterized test that runs over all the excluded extensions? Might make it easier to assert any changes in the future